apps/condo/domains/ticket/schema/TicketFile.js

```diff
@@ -23,7 +23,7 @@ const TicketFile = new GQLListSchema('TicketFile', {
     fields: {
         file: {
             schemaDoc: 'File object with meta information and publicUrl',
-            type: File,
+            type: FileWithUTF8Name,
```

Comment:
You should propably left string here, like so:
```suggestion
            type: 'File',
```

Reply:
And then map this string into your file field... inside `@open-condo/keystone`

Reply:
seems like logic with custom fields requires type as an object (like for LocalizedText or DateInterval etc)
Otherwise it thinks that it's a normal `File` and doesn't use `FileWithUTF8Name` logic

Reply:
edit: updated File mapping to use FileWIthUTF8Name